docs/pywbemcli/features.rst

```diff
@@ -95,21 +95,24 @@ DeleteQualifier                    Not Implemented
 Specifying CIM property and parameter values
 --------------------------------------------
 
-The :ref:`instance create command`, :ref:`instance modify command`, :ref:`class invokemethod command`, and
-:ref:`instance invokemethod command` define the values of properties and parameters that
-are to be applied to CIM instances and methods to be sent to the WBEM server.
+The :ref:`instance create command`, :ref:`instance modify command`,
+:ref:`class invokemethod command`, and :ref:`instance invokemethod command`
+define the values of properties and parameters that are to be applied to CIM
+instances and methods to be sent to the WBEM server.
 
-For a single property or parameter this is the ``--property``/``-p`` or
-``--parameter``/``-p`` option with the name and value in the form:
+For a single property or parameter these are the ``--property``/``-p`` or
+``--parameter``/``-p`` command options with its name and value in the form:
```

Comment:
Change "with its name and value" to "with name and value"

Reply:
DONE